cli/src/plugins/javadoc/index.ts

```diff
@@ -182,6 +183,57 @@ function makeSuperclassList(project: Project, doc: ParsedClassDoc) {
   ];
 }
 
+function makeInterfaceMethodList(project: Project, doc: ParsedClassDoc, interfaceTypes: Type[], inheritedMethods: {[k: string]: string[]}) {
+  if (interfaceTypes.length === 0) {
+    return [];
+  }
+  return [
+    md.heading(2, md.text("Methods Inherited from Interfaces")),
+    md.root(interfaceTypes.map((interfaceType) => {
+      return [
+        md.heading(3, md.text(interfaceType.qualifiedTypeName)),
+        md.list(
+          "unordered",
+          inheritedMethods[interfaceType.qualifiedTypeName].map((method) => {
+            return md.listItem(project.linkToEntity([interfaceType.qualifiedTypeName, method].join('.'), method))
+          }))
+      ]
+    }).flat(1))
+  ]
+}
+
+function makeSuperclassMethodList(project: Project, doc: ParsedClassDoc, superclasses: Type[], inheritedMethods: {[k: string]: string[]}) {
+  if (superclasses.length === 0) {
+    return [];
+  }
+  return [
+    md.heading(2, md.text("Methods Inherited from Superclasses")),
+    md.root(superclasses.map((superclass) => {
+      return [
+        md.heading(3, md.text(superclass.qualifiedTypeName)),
+        md.list(
+        "unordered",
+        inheritedMethods[superclass.qualifiedTypeName].map((method) => {
+          return md.listItem(project.linkToEntity([superclass.qualifiedTypeName, method].join('.'), method))
+        }))
+      ]
+    }).flat(1)
+    ),
+  ]
+}
+
+
+function makeInheritedMethodList(project: Project, doc: ParsedClassDoc) {
+  const { interfaceTypes, inheritedMethods, superclasses } = doc;
+  if (inheritedMethods === null || Object.keys(inheritedMethods).length === 0) {
```

Comment:
Avoid `=== null`. Null and undefined `==` null, and the property may be undefined as well as null.